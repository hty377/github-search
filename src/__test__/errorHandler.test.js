import { shallow } from "enzyme";
import ErrorHandle from "../components/ErrorHandler";

describe("Error", () => {
    it("should render Error Component", ()=>{
        const wrapper = shallow(
            <ErrorHandle error="No repositories found" />
        );
        expect(wrapper.getElements()).toMatchSnapshot();
    });
});