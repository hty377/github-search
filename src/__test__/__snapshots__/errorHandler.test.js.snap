// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render Error Component 1`] = `
Array [
  <p>
    No repositories found
  </p>,
]
`;
